GoogleForCreators/web-stories-wp
# Fixed tooltips for element links

## Context

This is an attempt at fixing the tooltips.

## To-do

- [x] Make URL icon rounded
- [x] Hide URL icon if missing
- [x] Add external link icon
- [x] Remote protocol if raw url displayed
- [x] Fix tail position - see comment below

## User-facing changes

| Has icon? | Element size | Before | After |
|-|-|-|-|
| ✅ | Wide | <img width="766" alt="Screen Shot 2022-12-19 at 14 54 52" src="https://user-images.githubusercontent.com/637548/208442688-9b5c80e9-95da-416e-a1c2-e5917e114072.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 41" src="https://user-images.githubusercontent.com/637548/208442610-812e85bd-ef8c-4d38-a429-0030671b0b0b.png"> |
| ✅ | Medium | <img width="766" alt="Screen Shot 2022-12-19 at 14 54 56" src="https://user-images.githubusercontent.com/637548/208442705-712aaa79-4d5c-4c41-9543-6613a5c487aa.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 45" src="https://user-images.githubusercontent.com/637548/208442624-02e0b5a3-7f29-411d-bead-d49cacdc9cab.png"> |
| ✅ | Narrow | <img width="766" alt="Screen Shot 2022-12-19 at 14 55 00" src="https://user-images.githubusercontent.com/637548/208442734-f306183d-c5b4-4349-b5ef-d95f6734683c.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 49" src="https://user-images.githubusercontent.com/637548/208442653-c4951905-ccef-4323-ad14-a16e095e6673.png"> |
| 🚫  | Wide | <img width="766" alt="Screen Shot 2022-12-19 at 14 55 07" src="https://user-images.githubusercontent.com/637548/208442583-32928009-ca22-455d-8e8f-398f3989d194.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 28" src="https://user-images.githubusercontent.com/637548/208442468-dfa443c8-9342-4d8e-bc0d-fc5e48767507.png"> |
| 🚫 | Medium | <img width="766" alt="Screen Shot 2022-12-19 at 14 55 10" src="https://user-images.githubusercontent.com/637548/208442558-24f26404-8175-4155-aa07-34028768325d.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 31" src="https://user-images.githubusercontent.com/637548/208442482-d36f058e-0ea3-4b15-a64c-7d93f5eb275b.png"> |
| 🚫 | Narrow | <img width="766" alt="Screen Shot 2022-12-19 at 14 55 14" src="https://user-images.githubusercontent.com/637548/208442534-9fc8163f-3c4a-4ef8-b4ed-925d62d246ca.png"> | <img width="766" alt="Screen Shot 2022-12-19 at 14 51 35" src="https://user-images.githubusercontent.com/637548/208442499-5fe7d36c-be84-4f8c-925e-8c79294dd6d0.png"> |

## Testing Instructions

This PR can be tested by following these steps:

1. Add elements with links in various sizes and positions
2. Observe the link hover preview

## Checklist




---

Fixes #12834
